packages/components/atoms/more-filter-atom/src/MoreFilter.atom.tsx

```diff
@@ -12,34 +12,31 @@ export const MoreFilter = ({
 	fontColor,
 	onClick,
 }: MoreFilterProps): JSX.Element => {
-	const [showMore, setShowMore] = React.useState(false);
-	const toggleClass = () => {
-		setShowMore(!showMore);
-	};
-	const moreOptions = {
-		height: 32,
-	};
 
 	const sampleData: { text: string; id: number }[] = [];
 	for (let index = 0; index < 7; index += 1) {
-		sampleData[index] = { text: 'shop by category', id: index };
```

Comment:
If you are gonna manually assign index then you probably don't need the for loop anymore.